src/main/java/service/validate/InputVerifier.java

```diff
@@ -0,0 +1,32 @@
+package service.validate;
+
+import exception.ErrorMessage;
+
+public class InputVerifier {
+
+    public static void validateNameLength(String[] names) {
+        for (String name : names) {
+            checkNameLength(name);
+            checkSpace(name);
+        }
+    }
+
+    private static void checkSpace(String name) {
+        if (name.contains(" ")) {
+            throw new IllegalArgumentException(ErrorMessage.CANT_CONTAIN_SPACE.getMessage());
```

Comment:
ErrorMessage이 단순 message만을 가지고 있는거 같아요.

이경우 enum을 지우고, message만을 사용하는게 코드를 좀더 명확하게 볼 수 있을 것 같아요

```suggestion
        if (name.contains(" ")) {
            throw new IllegalArgumentException("자동차 이름에 공백이 포함될 수 없습니다.");
```

Reply:
Enum부분도 베베와 상의 후 제거했습니다!